config/base/rbac/role.yaml

```diff
@@ -7,367 +9,250 @@ rules:
+  - cloud.istio.io
+  - config.istio.io
+  - istio.istio.io
+  - networking.istio.io
+  - scalingpolicy.istio.io
+  - security.istio.io
   resources:
-  - replicasets
+  - '*'
   verbs:
-  - get
-  - list
-  - watch
+  - '*'
 - apiGroups:
-  - policy
+  - authentication.k8s.io
   resources:
-  - poddisruptionbudgets
+  - tokenreviews
   verbs:
-  - get
-  - list
-  - watch
   - create
-  - update
-  - patch
-  - delete
 - apiGroups:
   - autoscaling
   resources:
   - horizontalpodautoscalers
   verbs:
-  - get
-  - list
-  - watch
   - create
-  - update
-  - patch
   - delete
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - '*'
-- apiGroups:
-  - rbac.authorization.k8s.io
-  resources:
-  - clusterroles
-  - clusterrolebindings
-  - roles
-  - rolebindings
-  - ""
-  verbs:
   - get
   - list
-  - watch
-  - create
-  - update
   - patch
-  - delete
-- apiGroups:
-  - authentication.k8s.io
-  resources:
-  - tokenreviews
-  verbs:
-  - create
+  - update
+  - watch
 - apiGroups:
   - certificates.k8s.io
   resources:
   - certificatesigningrequests
   - certificatesigningrequests/approval
   - certificatesigningrequests/status
   verbs:
-  - update
   - create
-  - get
   - delete
+  - get
+  - update
   - watch
 - apiGroups:
   - certificates.k8s.io
+  resourceNames:
+  - kubernetes.io/legacy-unknown
   resources:
   - signers
   verbs:
   - approve
 - apiGroups:
-  - extensions
-  - networking.k8s.io
+  - discovery.k8s.io
   resources:
-  - ingresses
+  - endpointslices
   verbs:
   - get
   - list
   - watch
 - apiGroups:
   - extensions
-  - networking.k8s.io
   resources:
-  - ingresses/status
-  verbs:
-  - '*'
-- apiGroups:
-  - networking.k8s.io
-  resources:
-  - ingressclasses
-  - ingresses
+  - deployments
   verbs:
   - get
   - list
   - watch
 - apiGroups:
-  - networking.k8s.io
+  - extensions
   resources:
-  - ingresses/status
+  - deployments/finalizers
   verbs:
-  - '*'
+  - update
 - apiGroups:
-  - networking.x-k8s.io
+  - extensions
   resources:
-  - '*'
+  - ingresses
+  - ingresses/status
   verbs:
-  - get
-  - list
-  - watch
+  - '*'
 - apiGroups:
-  - discovery.k8s.io
+  - extensions
   resources:
-  - endpointslices
+  - replicasets
   verbs:
   - get
   - list
   - watch
 - apiGroups:
-  - istio.banzaicloud.io
+  - extensions;networking.k8s.io
   resources:
-  - istios
-  - istios/finalizers
+  - ingresses
   verbs:
   - get
   - list
   - watch
-  - create
-  - update
-  - patch
-  - delete
 - apiGroups:
-  - istio.banzaicloud.io
+  - extensions;networking.k8s.io
   resources:
-  - istios/status
-  verbs:
-  - get
-  - update
-  - patch
-- apiGroups:
-  - authentication.istio.io
-  - cloud.istio.io
-  - config.istio.io
-  - istio.istio.io
-  - networking.istio.io
-  - security.istio.io
-  - scalingpolicy.istio.io
-  resources:
-  - '*'
+  - ingresses/status
   verbs:
   - '*'
 - apiGroups:
   - istio.banzaicloud.io
   resources:
-  - meshgateways
```

Comment:
Some rules seems to be missing from the edited rbac role based on the operator logs:
```
istio-operator-controller-manager-0 manager E0928 09:39:14.879766       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1beta1.RemoteIstio: remoteistios.istio.banzaicloud.io is forbidden: User "system:serviceaccount:istio-system:default" cannot list resource "remoteistios" in API group "istio.banzaicloud.io" at the cluster scope
istio-operator-controller-manager-0 manager E0928 09:39:22.648542       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1.ValidatingWebhookConfiguration: validatingwebhookconfigurations.admissionregistration.k8s.io is forbidden: User "system:serviceaccount:istio-system:default" cannot list resource "validatingwebhookconfigurations" in API group "admissionregistration.k8s.io" at the cluster scope
istio-operator-controller-manager-0 manager E0928 09:39:28.348041       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.6/tools/cache/reflector.go:125: Failed to list *v1beta1.MeshGateway: meshgateways.istio.banzaicloud.io is forbidden: User "system:serviceaccount:istio-system:default" cannot list resource "meshgateways" in API group "istio.banzaicloud.io" at the cluster scope
```